tests/test_logistic_regression_model_builder.py

```diff
@@ -0,0 +1,95 @@
+#*******************************************************************************
+# Copyright 2020 Intel Corporation
+# All Rights Reserved.
+#
+# This software is licensed under the Apache License, Version 2.0 (the
+# "License"), the following terms apply:
+#
+# You may not use this file except in compliance with the License.  You may
+# obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#*******************************************************************************
+
+import unittest
+import daal4py as d4p
+import numpy as np
+from sklearn.datasets import load_iris
+from sklearn.datasets import load_breast_cancer
+from sklearn.linear_model import LogisticRegression
+from daal4py.sklearn._utils import daal_check_version
+
+from daal4py import _get__daal_link_version__ as dv
+# First item is major version - 2021, second is minor+patch - 0110, third item is status - B
+daal_version = (int(dv()[0:4]), dv()[10:11], int(dv()[4:8]))
+version = ((2021, 'P', 110))
```

Comment:
Use it into to code. Because it file will can have other check version

Reply:
Corrected